AK/StringBase.h

```diff
@@ -69,9 +69,17 @@ class StringBase {
 
     [[nodiscard]] bool operator==(StringBase const&) const;
 
-    [[nodiscard]] ALWAYS_INLINE FlatPtr raw(Badge<FlyString>) const { return bit_cast<FlatPtr>(m_data); }
+    [[nodiscard]] ALWAYS_INLINE FlatPtr raw(Badge<FlyString>) const { return m_raw; }
```

Comment:
I don't understand how `m_raw` is useful here. Can't we use `m_data == nullptr`?